torch_scatter/utils.py

```diff
@@ -9,5 +9,5 @@ def broadcast(src: torch.Tensor, other: torch.Tensor, dim: int):
             src = src.unsqueeze(0)
     for _ in range(src.dim(), other.dim()):
         src = src.unsqueeze(-1)
-    src = src.expand_as(other)
+    src = src.expand_as(other.size())
```

Comment:
I guess this needs to be:
```suggestion
    src = src.expand(other.size())
```

Reply:
Sorry for the mistake, already fixed